src/app/screens/Publishers/index.tsx

```diff
@@ -14,7 +17,16 @@ function Publishers() {
     keyPrefix: "publishers",
   });
 
+  const { settings, updateSetting } = useSettings();
+
   const [publishers, setPublishers] = useState<Publisher[]>([]);
+  const tips = settings.tips || [];
+
+  const hasTips = tips.length > 0;
+
+  function hasTip(id: string) {
+    return tips.indexOf(id) > -1;
```

Comment:
Does [includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) work here?

Reply:
yeah, that reads better. done.